openssh.yaml

```diff
@@ -121,12 +121,17 @@ subpackages:
     description: "OpenSSH server"
     pipeline:
       - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/usr/sbin
+          mkdir -p "${{targets.subpkgdir}}"/usr/sbin "${{targets.subpkgdir}}"/var/empty
+          chmod 755 "${{targets.subpkgdir}}"/var/empty
```

Comment:
Could you add a test in this PR as well? You said the change fixes a behavior where the daemon doesn't come up, it would be great to reflect that in a test to safeguard against regression.

Reply:
Sure, good idea.  I added a test.